huaweicloud/services/dli/resource_huaweicloud_dli_flinkjar_job.go

```diff
@@ -27,6 +27,10 @@ import (
 // @API DLI PUT /v1.0/{project_id}/streaming/flink-jobs/{job_id}
 // @API DLI POST /v1.0/{project_id}/streaming/jobs/stop
 // @API DLI DELETE /v1.0/{project_id}/streaming/jobs/{job_id}
+// @API DLI DELETE /v1.0/{project_id}/streaming/jobs/{job_id}
```

Comment:
```suggestion
// @API DLI DELETE /v1.0/{project_id}/streaming/jobs/{job_id}
```
same reference for the delete API?

Reply:
fixed